3legacy/5.1/enterprise-5.1.5/overload.gradle

```diff
@@ -11,5 +11,4 @@ ext {
 
 dependencies {
     baseShareWar 'org.alfresco:share:5.1.5@war'
-    sharedLib(group: 'org.postgresql', name: 'postgresql', version: '9.4.1212')
```

Comment:
why don't we need the postgres driver anymore ?

Reply:
The postgres driver is already in there.
Legacy images build on top of repository images.
The postgres driver is already added in the repository image that is underneath.
The only thing that needs to be added in the legacy subproject is Share.

I checked the built image for presence of the driver.

```
$ docker run hub.xenit.eu/lvierbergen/alfresco-enterprise:5.1.5-build-DOCKER-322-32 stat /usr/local/tomcat/shared/lib/postgresql-9.4.1212.jar
> Executing /docker-entrypoint.d/80-java-opts.sh
Result: JAVA_OPTS=
> Executing /docker-entrypoint.d/85-init-tomcat.sh
Tomcat init start
Tomcat init done
> Executing /docker-entrypoint.d/89-check-validity.sh
> Executing /docker-entrypoint.d/90-init-alfresco.sh
Alfresco init start
Alfresco init done
> Executing /docker-entrypoint.d/91-init-share-customized.sh
Share init start
Share init done
> Starting CMD: stat /usr/local/tomcat/shared/lib/postgresql-9.4.1212.jar
> exec stat /usr/local/tomcat/shared/lib/postgresql-9.4.1212.jar
  File: /usr/local/tomcat/shared/lib/postgresql-9.4.1212.jar
  Size: 680445    	Blocks: 1336       IO Block: 4096   regular file
Device: 77h/119d	Inode: 19690087    Links: 1
Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2020-05-07 12:08:22.000000000 +0000
Modify: 2020-05-07 12:08:22.000000000 +0000
Change: 2020-05-07 13:50:07.012357816 +0000
 Birth: -
```